rerun_cpp/src/rerun/recording_stream.hpp

```diff
@@ -129,33 +130,11 @@ namespace rerun {
         ///
         /// ## Parameters
         ///
-        /// port:
```

Comment:
This should make a mention of `SpawnOptions` and once we have a doc system ideally link to it?